src/components/ClipboardButton/FakeButton.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {MouseEventHandler} from 'react';
+import {block} from '../utils/cn';
+import {useActionHandlers} from '../utils/useActionHandlers';
+import {ClipboardIcon} from '../ClipboardIcon';
+import {CopyToClipboardStatus} from '../CopyToClipboard';
+import {ClipboardButtonProps} from './ClipboardButton';
+import './ClipboardButton.scss';
+
+const b = block('clipboard-button');
+
+const DEFAULT_ICON_SIZE = 24;
+
+interface FakeButtonProps extends Pick<ClipboardButtonProps, 'className' | 'size' | 'qa'> {
+    status: CopyToClipboardStatus;
+    onClick?: MouseEventHandler<HTMLSpanElement>;
+}
+
+export function FakeButton({
+    status,
+    className,
+    qa,
+    size = DEFAULT_ICON_SIZE,
+    onClick,
```

Comment:
Where are we passing onClick prop? I cannot see.

Reply:
It is passed by `react-copy-to-clipboard` via cloning element

Reply:
It is passed by `react-copy-to-clipboard` via cloning element

Reply:
Here

https://github.com/nkbt/react-copy-to-clipboard/blob/fa8b7e2b1cfdcda5908b2d30f91f02c6dd008aba/src/Component.js#L58